python/mxnet/_numpy_op_doc.py

```diff
@@ -450,3 +450,67 @@ def _np_reshape(a, newshape, order='C', out=None):
     ndarray.reshape : Equivalent method.
     """
     pass
+
+
+def _np_roll(a, shift, axis=None):
+    """Roll array elements along a given axis.
```

Comment:
Add function signature `_np_roll(a, shift, axis=None)` in the first line and start doc description two lines below.